sass/components/_react-select.scss

```diff
@@ -62,14 +54,17 @@
         }
 
         &:hover {
-            border-color: v(center-channel-color-40);
-
             .react-select__indicator {
                 opacity: .8;
             }
         }
     }
 
+    .react-select__control--is-focused {
+        box-shadow: inset 0 0 0 1px v(button-bg);
+        border-color: v(button-bg);
```

Comment:
can we use the rgba variables too? it's part of the migration to compass design system.